pkg/testcase/secretsencrypt.go

```diff
@@ -30,10 +29,8 @@ func TestSecretsEncryption() {
 	secretsEncryptOps("rotate", product, cpIp, nodes)
 	secretsEncryptOps("reencrypt", product, cpIp, nodes)
 
-	if strings.Contains(os.Getenv("TEST_TYPE"), "both") {
-		shared.LogLevel("info", "TEST: 'NEW' Secrets Encryption method")
-		secretsEncryptOps("rotate-keys", product, cpIp, nodes)
-	}
+	shared.LogLevel("info", "TEST: 'NEW' Secrets Encryption method")
```

Comment:
Do we need to keep this? as this is the only method that is supported ATM?

Reply:
it helps to segregate the output - when u read the logs - like both the reencrypt and rotate-keys outputs look similar.. you can search for the TEST keyword and go back and forth. its just for easier traversal of the logs. I can remove if it is not desirable.

Reply:
it was easier to search through logs - since both reencrypt and rotate-keys outputs look exactly the same. But I can remove this.

Reply:
removed